src/Auth.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react"
+
+import Signin from "./signin";
+import Signup from "./signup";
+
+
+
+class Auth extends React.Component {
```

Comment:
what does this class do?

Reply:
the page i referred always asked to have a auth page separate for login signup and i was seeing that

for better native response when u switch

Reply:
u can have like common stuff only for those  2 pages signin and signup, like a side welcome page i will explain that in a call

Reply:
Lets go with it

Reply:
okay cool